lib/Basic/Statistic.cpp

```diff
@@ -59,6 +57,16 @@ bool environmentVariableRequestedMaximumDeterminism() {
   return false;
 }
 
+uint64_t getInstructionsExecuted() {
```

Comment:
Nitpick: It's nice to write `swift::` explicitly to ensure it matches the header declaration.